app/directives/forms/edit/reservation.js

```diff
@@ -338,6 +338,15 @@ define(['app', 'lodash',
                             });
                         } //init
 
+                        //============================================================
+                        //
+                        //============================================================
+                        $scope.setVideoUrl= function(doc) {
+                            if(doc.video)
+                              doc.videoUrl=$scope.room.videoUrl
```

Comment:
Leave the field tot he value if the <input>. Just trim it. if the value is empty the UI will use the value from the room instead.

Reply:
It is prefilled with the value from the room already if it exists.   That why I thought a placeholder was not needed.  But if the room is not set up yet it will need a placeholder.  If that is case it will need valid url check no?

Reply:
@randyhoulahan Don't prefill leave empty. and room will be the fallback url